server/Dockerfile

```diff
@@ -47,9 +47,13 @@ WORKDIR /scalardb/server
 COPY log4j2.properties .
 # Support use cases where environment variables are used to configure the database
 COPY database.properties.tmpl .
+COPY docker-entrypoint.sh .
 
+ENV JAVA_OPT_MAX_RAM_PERCENTAGE '50.0'
```

Comment:
I think we can increase the default value to `70.0`? Or is there any problem if we increase it?

Reply:
The reason for this default value is that `scalardb-server` uses a `relatively` small heap size.
Corrections have been made to this issue that you pointed out. :bow:

https://github.com/scalar-labs/scalardb/pull/392/commits/6965e64268ec3735a54307b8ec5e0c128eae9ff1